src/DynamoCore/Logging/CrashReportTool.cs

```diff
@@ -62,36 +62,76 @@ internal enum MINIDUMP_TYPE
             MiniDumpValidTypeFlags = 0x01ffffff
         };
 
+        [DllImport("kernel32.dll")]
+        static extern IntPtr GetCurrentProcess();
```

Comment:
can you explain why you need to use these via pinvoke instead of using the managed versions via the `Process` object?

Reply:
The managed Process object is good enough (matches the native process info)
The thread info however does not. I need to use the native Thread information (ex threadId) because managed threads (created by clr) do not match the native (OS) threads.
[Info](https://docs.microsoft.com/en-us/dotnet/api/system.threading.thread?redirectedfrom=MSDN&view=net-6.0#getting-information-about-and-controlling-threads)